<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SALIK" failures="1" tests="4" name="RunMethodSignin" time="22.557" errors="0" timestamp="2025-04-15T21:15:17 IST">
  <testcase classname="com.frameworks.projectCMS.testcases.SigninTest" name="testWithInvalidEmail" time="0.966"/>
  <testcase classname="com.frameworks.projectCMS.testcases.SigninTest" name="navigateToforgotPasswordPage" time="0.572"/>
  <testcase classname="com.frameworks.projectCMS.testcases.SigninTest" name="testSuccessfulllogin" time="1.763"/>
  <testcase classname="com.frameworks.projectCMS.testcases.SigninTest" name="checkLoginLoadTime" time="1.851">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.frameworks.projectCMS.pages.SigninPage.calculateLoginTime(SigninPage.java:85)
at com.frameworks.projectCMS.testcases.SigninTest.checkLoginLoadTime(SigninTest.java:34)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- checkLoginLoadTime -->
</testsuite> <!-- RunMethodSignin -->
